Support for mongodb 4.x.x (types)

I am using `mongo-seeding` and `mongoose` in the same (typescript) project. I recently upgrade `mongoose` to version  6.0.4. This version of `mongoose` depends on `mongodb` 4.1.1 which does not play nicely with `mongo-seeding` right now.

```typescript
node_modules/mongo-seeding/dist/config.d.ts:3:10 - error TS2305: Module '"mongodb"' has no exported member 'CollectionInsertManyOptions'.

3 import { CollectionInsertManyOptions, MongoClientOptions } from 'mongodb';
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/mongo-seeding/dist/database/database.d.ts:1:14 - error TS2305: Module '"mongodb"' has no exported member 'CollectionInsertManyOptions'.

1 import { Db, CollectionInsertManyOptions, MongoClient } from 'mongodb';
               ~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/mongo-seeding/dist/database/database.d.ts:32:167 - error TS2694: Namespace 'node_modules/mongodb/mongodb"' has no exported member 'InsertWriteOpResult'.

32     insertDocumentsIntoCollection(documentsToInsert: any[], collectionName: string, collectionInsertOptions?: CollectionInsertManyOptions): Promise<import("mongodb").InsertWriteOpResult<any>>;

                         ~~~~~~~~~~~~~~~~~~~

node_modules/mongo-seeding/dist/database/database.d.ts:55:93 - error TS2694: Namespace '"node_modules/mongodb/mongodb"' has no exported member 'DeleteWriteOpResultObject'.

55     removeAllDocumentsIfCollectionExists(collectionName: string): Promise<import("mongodb").DeleteWriteOpResultObject | undefined>;
                                                                                               ~~~~~~~~~~~~~~~~~~~~~~~~~

```

Do you plan to upgrade mongodb (type) dependencies in the foreseeable future?

--- a/core/package.json
+++ b/core/package.json
@@ -54,12 +54,12 @@
     "debug": "^4.3.1",
     "extend": "^3.0.0",
     "import-fresh": "^3.3.0",
-    "mongodb": "^3.6.4"
+    "mongodb": "^3.6.4",
+    "@types/mongodb": "^3.6.20"
   },
   "devDependencies": {
     "@types/extend": "^3.0.1",
     "@types/jest": "^27.0.2",
-    "@types/mongodb": "^3.6.20",
     "jest": "^27.2.5",
     "ts-jest": "^27.0.6",
     "typescript": "^4.4.4"

